src/shell/lisa_shell

```diff
@@ -143,6 +143,7 @@ case "x${CMD^^}" in
 	_lisa-update-submodules
 	;;
 'xALL')
+	_lisa-update-submodules
```

Comment:
I would rather not do that.  `_lisa-update-all`  calls `_lisa-update-submodules` at the end on purpose.  Instead of calling `_lisa-update-submodules` both before and at the end of `_lisa-update-all`, let's fix the misleading error: the fact that `_lisa-update-all` tells you there are outstanding uncommitted changes when in fact there aren't.  This happens because `git diff-index` takes submodules into account, when for what we want it shouldn't.  We should modify the line to read `git diff-index --quiet --ignore-submodules HEAD`.